hack/update-translations.sh

```diff
@@ -71,7 +72,7 @@ if [[ "${generate_pot}" == "true" ]]; then
   perl -pi -e 's/\\\)/\\\\\)/g' tmp.pot
   kube::util::ensure-temp-dir
   if msgcat -s tmp.pot > "${KUBE_TEMP}/template.pot"; then
-    mv "${KUBE_TEMP}/template.pot" translations/kubectl/template.pot
+    mv "${KUBE_TEMP}/template.pot" "${TRANSLATIONS}/kubectl/template.pot"
```

Comment:
nit, if this is already under `staging/src/k8s.io/kubectl`, can drop the `kubectl` dir under here

Reply:
Good catch. Will change that.

Reply:
On second thought, how about keeping the directory structure as it is now?

I am worried about changing many things at once here. Maybe this optimization can come later on?